workspaces/analytics/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
if it is in the workspace, it should inherit the workspace prettierrc correct?

Reply:
yeah it started separated. cleaning it now